src/main.rs

```diff
@@ -40,6 +40,10 @@ enum Commands {
         #[arg(help = "Distorted video", value_hint = clap::ValueHint::FilePath)]
         distorted: PathBuf,
 
+        /// How many worker threads to use for decoding & calculating scores.
+        #[arg(long, short)]
+        threads: Option<usize>,
```

Comment:
I want to add support for other methods of threading that don't add memory usage impact. So maybe we can call this parameter "frame_threads" or something to indicate that this is specifically the number of frames to run in parallel? And note in the help text that this will increase memory usage.